### ISSUE TITLE
"Log file: Not available" with JDK11

### ISSUE THREAD

**Issue by sergtop** [RWTH-i5-IDSG/steve#90]
Recent version with JDK11 can`t start logging due to 

WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.

switching back to 1.8 in pom.xml solves issue

**Comment by goekay** [comment]
can reproduce the warnings. but the log file is there. it does start logging. are you sure about this?

**Comment by sergtop** [comment]
yes, the file is created, but it can`t be accessed via */manager/log URI from webservice

**Comment by goekay** [comment]
oh okay, thanks, i will look into it.

**Comment by goekay** [comment]
i want to mention, that the warning is still there. but reading the configuration where to log file is is now successful. 

there are some other warnings related to jooq as well. these should be fixed by the library maintainers with future releases.
